### PR TITLE
prod | GIDK-3nfxzv | Aditi | new list design changes

### PR DESCRIPTION
**Pull request by aditi-tiwari17**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
https://app.clickup.com/t/3nfxzv

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/contact/aging-report/aging-report.component.ts [hunk 1]
```diff
@@ -334,6 +363,111 @@ export class AgingReportComponent implements OnInit, OnDestroy {
...
+    }
+
+    /**
+     * Toogles the search field
+     *
+     * @param {string} fieldName Field name to toggle
+     * @param {*} el Element reference for focusing
+     * @memberof AgingReportComponent
+     */
+    public toggleSearch(fieldName: string, el: any) {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/aging-report/aging-report.component.ts:448]: add return type

File: apps/web-giddh/src/app/contact/contact.component.html [hunk 2]
```diff
@@ -101,129 +179,118 @@
...
+                                    <button *ngIf="selectedCheckedContacts?.length" id="button-basic" dropdownToggle type="button"
+                                        class="btn btn-default dropdown-toggle mr-0" aria-controls="dropdown-basic">
+                                        <span class="icon-arrow-down font-12"></span>
                                     </button>
-
+                                    <ul id="dropdown-basic" *dropdownMenu class="dropdown-menu pd-l1 pd-r1" role="menu"
+                                        aria-labelledby="button-basic">
+                                        <li role="menuitem" (click)="openEmailDialog();">
+                                            {{commonLocaleData?.app_email}}</li>
+                                        <li role="menuitem" (click)="openSmsDialog();">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.html:223]: this should be commented

File: apps/web-giddh/src/app/contact/contact.component.ts [hunk 3]
```diff
@@ -1446,4 +1458,55 @@ export class ContactComponent implements OnInit, OnDestroy {
...
+     * Returns the placeholder for the current searched field
+     *
+     * @param {string} fieldName Field name for which placeholder is required
+     * @returns {string} Placeholder text
+     * @memberof ContactComponent
+     */
+    public getSearchFieldText(fieldName: string): string {
+        if (fieldName === 'name') {
+            return this.localeData?.search_name;
+        }
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/contact/contact.component.ts:1510]: add return type for unmatching if condition

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 4]
```diff
@@ -168,93 +242,116 @@
...
+                        <div class="col-sm-6 col-md-6 mt-2">
+                            <button class="btn btn-sky-blue">{{commonLocaleData?.app_submit}}</button>
+                        </div>
                     </div>
-                </ng-template>
-
+                </div>
+            </ng-template>
+
+                <div class="top-content">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html:320]: for this new html

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 5]
```diff
@@ -1,58 +1,134 @@
...
+                        <button *ngIf="!isCompany" id="ddl-newinv" dropdownToggle type="button"
+                            class="btn btn-sky-blue dropdown-toggle dropdown-btn-new"
+                            aria-controls="dropdown-ddl-newinv">
+                            {{localeData?.new_invoice}}
+                            <span class="caret"></span>
+                        </button>
+                        <ul id="dropdown-ddl-newinv" *dropdownMenu class="dropdown-menu" role="menu"
+                            aria-labelledby="ddl-newinv">
+                            <li role="menuitem">
+                                <a class="dropdown-item" href="javascript: void 0"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html:22]:
```suggestion
                                <a class="dropdown-item" href="javascript:;"
```

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 6]
```diff
@@ -1,58 +1,134 @@
...
+                                    (click)="goToInvoice('cash')">{{localeData?.cash_invoice}}</a>
+                            </li>
+                            <li role="menuitem">
+                                <a class="dropdown-item" href="javascript: void 0"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html:26]:
```suggestion
                                <a class="dropdown-item" href="javascript:;"
```

File: apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html [hunk 7]
```diff
@@ -41,31 +41,78 @@
...
                 </div>
+                <div class="advance-icon cursor-pointer btn btn-primary mr-0" [tooltip]="commonLocaleData?.app_advance_search"
+                            (click)="openModal()">
+                            <span class="pd-r05 d-sm-flex d-none">{{commonLocaleData?.app_filter}}</span>
+                            <span class="icon-advance-filter"></span>
+                        </div>
                 <!-- Invoice filter section ends -->
             </div>
 
+            <div class="top-content">
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html:60]: for this new html

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/contact/aging-report/aging-report.component.ts (hunk 1 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/contact/aging-report/aging-report.component.ts:448]:
```suggestion
    public toggleSearch(fieldName: string, el: any): void {
```

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 8]
```diff
@@ -71,153 +109,195 @@
...
-
-                        <!-- Section to create New Sales Invoice, Cash Invoice, Credit Note and Debit Note -->
-                        <div class="export-button-wrap" *ngIf="selectedVoucher === 'sales'">
-                            <button class="btn btn-primary"
-                                [disabled]="((selectedInvoices && selectedInvoices.length > 0) ? selectedInvoices.length : voucherData && voucherData.totalItems) === 0"
-                                (click)="openModal(bulkExport)">{{commonLocaleData?.app_export}}</button>
-                        </div>
-                        <div class="btn-group desktop-view-btn" dropdown *ngIf="!isCompany && selectedVoucher === 'sales'"
-                            [autoClose]="true">
-                            <button *ngIf="!isCompany" id="ddl-newinv" dropdownToggle type="button"
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html:91]: Do check this ngIf should not be removed as we don't allow the user to create or edit in company mode

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html [hunk 9]
```diff
@@ -71,153 +109,195 @@
...
-                                    <a class="dropdown-item" href="javascript: void 0"
-                                        (click)="goToInvoice('cash')">{{localeData?.cash_invoice}}</a>
-                                </li>
-                                <li role="menuitem">
-                                    <a class="dropdown-item" href="javascript: void 0"
-                                        (click)="goToInvoice('sales')">{{localeData?.sales_invoice}}</a>
-                                </li>
-                            </ul>
-                        </div>
-                        <div *ngIf="selectedVoucher === 'purchase'" class="btn-wrapper">
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.html:109]: Do check this code block is getting removed for mobile screen

**Review by ravinderwalkover — APPROVED** [review]

**Review by prateek-giddh — COMMENTED** [review]
Please check

**Review by prateek-giddh — CHANGES_REQUESTED** [review]
Please check comment https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App/pull/8795#discussion_r669333024

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**